docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -535,6 +536,10 @@ Extract a path out into its own ingres if you need to isolate a certain path.
 * `nginx.ingress.kubernetes.io/global-rate-limit-window`: Configures a time window (i.e `1m`) that the limit is applied. Required.
 * `nginx.ingress.kubernetes.io/global-rate-limit-key`: Configures a key for counting the samples. Defaults to `$remote_addr`. You can also combine multiple NGINX variables here, like `${remote_addr}-${http_x_api_client}` which would mean the limit will be applied to requests coming from the same API client (indicated by `X-API-Client` HTTP request header) with the same source IP address.
 * `nginx.ingress.kubernetes.io/global-rate-limit-ignored-cidrs`: comma separated list of IPs and CIDRs to match client IP against. When there's a match request is not considered for rate limiting.
+* `nginx.ingress.kubernetes.io/global-rate-limit-set-headers`: Set to `true` to enable the addition of global rate limiting headers. Default is `false`
+  * x-rate-limit-limit: The maximum number of allowed requests in the time window;
+  * x-rate-limit-remaining: The number of remaining requests in the current window;
+  * x-rate-limit-window: The number of seconds of the period window;
```

Comment:
According to https://tools.ietf.org/id/draft-polli-ratelimit-headers-00.html#rfc.section.1.3

> RateLimit-Limit: containing the requests quota in the time window;
> RateLimit-Remaining: containing the remaining requests quota in the current window;
> RateLimit-Reset: containing the time remaining in the current window, specified in seconds or as a timestamp;

can you be consistent with this? I see that you use `x-rate-limit` instead of `ratelimit` and use `window` instead of `reset`. In other words, based on the linked proposal here are what I would expect to see:

```
RateLimit-Limit
RateLimit-Remaining
RateLimit-Reset
```

Reply:
I'd also link to https://tools.ietf.org/id/draft-polli-ratelimit-headers-00.html in the docs

Reply:
Updated naming for rate-limiting headers